### PR TITLE
GAP-2433: View last updated by (Applications)

### PR DESCRIPTION
**Pull request by ryan-tco**
## Description

Ticket # and link [GAP-2433](https://technologyprogramme.atlassian.net/jira/software/projects/GAP/boards/216?selectedIssue=GAP-2433)

Summary of the changes and the related issue. List any dependencies that are required for this change:

Moves the link into the application form onto the application form name.
Changes display of field based on published criteria.
Retrieves email of "Last Updated By" grant admin ID to display in table.

## Type of change

Please check the relevant options.

- [x] New feature (non-breaking change which adds functionality).

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes:

- [x] Unit Test

## Screenshots (if appropriate):

Screenshot 2024-03-06 at 09 13 49

Please attach screenshots of the change if it is a UI change:

# Checklist:

- [x] If I have listed dependencies above, I have ensured that they are present in the target branch.
- [x] I have performed a self-review of my code.
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation where applicable.
- [ ] I have ran cypress tests and they all pass locally.

[GAP-2433]: https://technologyprogramme.atlassian.net/browse/GAP-2433?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dominicwest — DISMISSED** [review]

File: packages/admin/src/pages/scheme/[schemeId]/index.test.tsx [hunk 1]
```diff
@@ -111,6 +118,11 @@ describe('scheme/[schemeId]', () => {
       mockedFindApplicationFormFromScheme.mockResolvedValue([
         applicationFormFoundStats(),
       ]);
+      (getApplicationFormSummary as jest.Mock).mockResolvedValue(
```
Comment (jgunnCO) [inline packages/admin/src/pages/scheme/[schemeId]/index.test.tsx:121]: can do the `jest.mocked` trick with this as well

File: packages/admin/src/pages/scheme/components/SchemeApplications.tsx [hunk 2]
```diff
@@ -3,19 +3,22 @@ import getConfig from 'next/config';
...
 }: SchemeApplicationsProps) => {
-  const formatDate = (date: string) => {
-    return new Date(date).toLocaleDateString('en-GB', {
-      day: 'numeric',
-      month: 'long',
-      year: 'numeric',
-    });
-  };
+  const formatDate = (timestamp: string) =>
+    `${moment(timestamp).format('D MMMM YYYY, h:mm a')}`;
```
Comment (john-tco) [inline packages/admin/src/pages/scheme/components/SchemeApplications.tsx:15]: Figma is incorrect, needs to look like this

```suggestion
    `${moment(timestamp).format('D MMMM YYYY, h:mma')}`;
```

File: packages/admin/src/pages/scheme/[schemeId]/index.test.tsx [hunk 3]
```diff
@@ -224,6 +220,9 @@ describe('scheme/[schemeId]', () => {
...
```
Comment (john-tco) [inline packages/admin/src/pages/scheme/[schemeId]/index.test.tsx:223]: can we use `jest.mocked` here

File: packages/admin/src/pages/scheme/components/SchemeApplications.tsx [hunk 4]
```diff
@@ -186,6 +188,7 @@ type SchemeApplicationsProps = {
   applicationForm: ApplicationFormSummary;
   applicationFormStats: FindApplicationFormStatsResponse;
   schemeVersion?: string;
+  editorOrPublisherEmail?: string;
```
Comment (john-tco) [inline packages/admin/src/pages/scheme/components/SchemeApplications.tsx:191]: can this ever be false?

```suggestion
  editorOrPublisherEmail: string;
```

**Review by john-tco — APPROVED** [review]

**Pull request merged**